api/s2n.h

```diff
@@ -871,6 +871,23 @@ extern int s2n_config_add_pem_to_trust_store(struct s2n_config *config, const ch
 S2N_API
 extern int s2n_config_wipe_trust_store(struct s2n_config *config);
 
+typedef enum { S2N_VERIFY_AFTER_SIGN_DISABLED, S2N_VERIFY_AFTER_SIGN_ENABLED } s2n_verify_after_sign;
+
+/**
+ * Toggle signature verification.
```

Comment:
```suggestion
 * Toggle signature verification after signing.
```
Should this also be changed with the changes from 0bfc9202847d809f50324df3bc14aec57e540e11?